### ISSUE TITLE
Notification sidebar is not accessible via keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1773]
On https://cryptpad.fr/notifications/ , the sidebar is not accessible for keyboard users.
